nengo_spa/modules/associative_memory.py

```diff
@@ -52,6 +52,8 @@ class AssociativeMemory(Network):
         See `nengo.Network` for more details.
     vocabs : VocabularyMap, optional
         Maps dimensionalities to the corresponding default vocabularies.
+    selection_net_args : dict
+        Additional arguments that will passed to the *selection_net*.
```

Comment:
IMO it would be clearer to name this `**selection_net_kwargs` with `kwargs` instead of `args`, and add `**` to disambigutate this from the case where the parameter could itself be a dictionary. And then say "Additional keyword arguments..."

Example here: https://arvoelke.github.io/nengolib-docs/nengolib.signal.nrmse.html#nengolib.signal.nrmse

Reply:
Good suggestion, I made the changes for `selection_net_args` and grepped for `kwargs` to have it consistent.